docker/docker-compose.yml

```diff
@@ -1,26 +1,34 @@
-version: '2'
+version: '3.2'
 
 services:
+
+### OMS SERVICE REGISTRY        #######################################
     omsserviceregistry:
+        build:
+            context: ./$PATH_OMS_REGISTRY/omsserviceregistry/
+            dockerfile: ./Dockerfile.dev
         image: aegee/omsserviceregistry:dev
         ports:
             - "7000:7000"
         volumes:
-            - ./oms-serviceregistry:/usr/src/app
-            - /var/run/docker.sock:/var/run/docker.sock:ro
-            - shared:/usr/src/shared
-        links:
-            - mongodb
-
-    mongodb:
-        image: mongo:latest
-        expose:
-            - "27017"
-        volumes:
-            - mongodb:/data/db
+            - type: bind
+              source: ./$PATH_OMS_REGISTRY/../
+              target: /usr/src/app
+            - type: bind
+              source: /var/run/docker.sock
+              target: /var/run/docker.sock
+              read_only: true
+            - type: volume
+              source: shared
+              target: /usr/src/shared
```

Comment:
I think this would be a perfect moment to change folder namings to the standards we came up with:
https://oms-project.atlassian.net/wiki/spaces/GENERAL/pages/24219843/Standardized+folder+namings